File: 프로그래머스/lv0/120882. 등수 매기기/등수 매기기.java
class Solution {
    public int[] solution(int[][] score) {
        int len = score.length, cnt=1;
        int[] answer = new int[len];
        double[] avg = new double[len];
        
        for(int i=0; i<len; i++){
            avg[i] = (score[i][0]+score[i][1])/2.0;
        }
        for(int i=0; i<len; i++){
            answer[i] = len+1;
            for(int j=0; j<len; j++){
                if(avg[i] >= avg[j])
                    answer[i]--;
            }
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120882. 등수 매기기/등수 매기기.js
function solution(score) {
    score = score.map((e)=> e[0]+e[1]);
    var score_sort = [...score].sort((a,b)=> b-a);
    var answer = [];
    score.map((e) => {
        answer.push(score_sort.indexOf(e)+1);
    })
    return answer;
}